packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.base.tsx

```diff
@@ -89,7 +89,7 @@ export class ChoiceGroupBase extends React.Component<IChoiceGroupProps, IChoiceG
       : (this.props as any)['aria-labelledby'];
 
     return (
-      <div role={role} className={classNames.applicationRole}>
+      <div role={role} className={classNames.applicationRole} id={this._id}>
```

Comment:
It might look like `<div role={role} className={classNames.applicationRole} {...nativeProps} >`. Check how other components do it.

Reply:
I've converted this to the native props in the latest commit